common/src/main/java/earth/terrarium/hermes/utils/ElementParsingUtils.java

```diff
@@ -116,7 +117,49 @@ public static Alignment parseAlignment(Map<String, String> parameters, String ke
         return defaultValue;
     }
 
-    public static <A,B, C> @Nullable C parsePair(
+    public static Map<String, String> parseStyleAttribute(Map<String, String> parameters) {
+
+        // Allow for using 'style="..."' attribute values to compactly specify multiple attributes.
+        // These compact attributes may be in 'key:value' or boolean form.
+        // Booleans can be negated with a leading '!'; both forms may have short names or synonyms.
+        // Example: style="!obfuscated,b,em,color:lavender,bg:0 #4a38d0"
+
+        Map<String, String> result = new HashMap<>();
+        String[] specificers = parameters.get("style").split(";");
+        String specValue;
+        for (String specifier : specificers) {
+            String[] specParts = specifier.split(":",2);
+            if (specParts.length == 2) {
+                // key:value form
+                specifier = specParts[0].strip();
+                specValue = specParts[1].strip();
+            } else {
+                // boolean form
+                specifier = specifier.strip();
```

Comment:
it should throw an exception if specParts length is greater than 2 as right now it will infer as a boolean if you for example do "color::" it will convert that to color = "true" which will cause errors later on which will confuse the user

Reply:
I didn't foresee this error, you got me there xD.

Having checked it's behavior now though, I think it's better as is--the description will still render and the mistake\* is more immediate, visual feedback (including "where" the problem is) than flipping over to the log.

IDK... is there a way to do both: log the syntactical error _and_ let the description render proceed?

(\* if it's a "color" spec, the resulting color is black; similarly parsing "background", "border", "align", "valign" as "true" booleans will also render in ways that are visually obvious.)